### PR TITLE
renamed, documented and slightly refactored lbvector functions

### PR DESCRIPTION
**Pull request by Ka-lua**
I gave those func_1234asdf functions some descriptive names and renamed some variables, but someone has to check if it OK compiles because I still can't build the project properly (even in vanilla state).

I renamed all occurences of those functions in the *s, *.c and *.h, I hope that's all that has to be done for renaming functions.

### CODE REVIEW

**Comment by Ka-lua** [comment]
The PR doesn't OK yet (my build setup works now after downgrading to devkitPPC r39), I'll update the PR with a working version later.

File: asm/melee/cm/camera.s [hunk 1]
```diff
@@ -9121,7 +9121,7 @@ func_80030BBC:
 /* 80030BF0 0002D7D0  83 E5 00 28 */	lwz r31, 0x28(r5)
 /* 80030BF4 0002D7D4  38 A1 00 28 */	addi r5, r1, 0x28
 /* 80030BF8 0002D7D8  38 7F 00 00 */	addi r3, r31, 0
-/* 80030BFC 0002D7DC  4B FD D6 15 */	bl func_8000E210
+/* 80030BFC 0002D7DC  4B FD D6 15 */	bl lbvector_project_world_to_screen_coord
```
Comment (PsiLupan) [inline asm/melee/cm/camera.s:9124]: Let's stick to shorter names and convention of something like:
lbvector_WorldToScreen

File: asm/melee/cm/camera.s [hunk 2]
```diff
@@ -9219,7 +9219,7 @@ func_80030CFC:
 /* 80030D44 0002D924  38 81 00 50 */	addi r4, r1, 0x50
 /* 80030D48 0002D928  38 BE 00 1C */	addi r5, r30, 0x1c
 /* 80030D4C 0002D92C  38 C1 00 38 */	addi r6, r1, 0x38
-/* 80030D50 0002D930  4B FD DA E9 */	bl func_8000E838
+/* 80030D50 0002D930  4B FD DA E9 */	bl lbvector_some_camera_vector_computation
```
Comment (PsiLupan) [inline asm/melee/cm/camera.s:9222]: I'm not sure calling this a camera vector computation is any more descriptive and is potentially misleading if it's used elsewhere.

Just stick to lbvector_8000E838

File: asm/melee/ft/chara/ftfox.s [hunk 3]
```diff
@@ -2624,7 +2624,7 @@ lbl_800E792C:
 /* 800E793C 000E451C  41 82 00 2C */	beq lbl_800E7968
 /* 800E7940 000E4520  38 7D 01 54 */	addi r3, r29, 0x154
 /* 800E7944 000E4524  38 9F 00 80 */	addi r4, r31, 0x80
-/* 800E7948 000E4528  4B F2 5E 49 */	bl func_8000D790
+/* 800E7948 000E4528  4B F2 5E 49 */	bl lbvector_angle_xy
```
Comment (PsiLupan) [inline asm/melee/ft/chara/ftfox.s:2627]: Let's try
lbvector_AngleXY

**Review by PsiLupan — CHANGES_REQUESTED** [review]
Left some general comments on the style. I directed it at specific instances, but I mean for them to apply to everything.

**Pull request merged**